Source/Core/Core/HW/DSPHLE/UCodes/UCodes.cpp

```diff
@@ -87,12 +87,12 @@ UCodeInterface* UCodeFactory(u32 crc, DSPHLE* dsphle, bool wii)
 	default:
 		if (wii)
 		{
-			PanicAlert("DSPHLE: Unknown ucode (CRC = %08x) - forcing AXWii.\n\nTry LLE emulator if this is homebrew.", crc);
+			PanicAlertT("DSPHLE: Unknown ucode (CRC = %08x) - forcing AXWii.\n\nTry LLE emulator if this is homebrew.", crc);
```

Comment:
This should probably be reorganized to put the description of what we expect the user to do before the technical details.